README.md

```diff
@@ -201,4 +220,4 @@ my items are:
 
 
 
-Leonardo Correa
+Leonardo Correa
```

Comment:
why this ?

Reply:
no changes done on this line , not sure why it showing as same line removed & added